ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/OAuthRedirectExample/README.md

```diff
@@ -0,0 +1,121 @@
+# OAuth Redirect Example
+
+This sample demonstrates an approach to using OAuth2 for authenticating with an ArcGIS portal.
+Unlike the pattern used in other samples where an `AuthenticationView` displays a log-in screen 
+using a webview within the app itself, this example sends an authorization request to an external
+browser along with a redirect Uri. The user can then authenticate securely in the browser 
+(using their ArcGIS.com credentials) and upon successful authentication, the redirect Uri 
+is notified with an authorization code to allow the log-in process to continue.
+
+In order for the call-back to the redirect Uri to be handled by the app, it is necessary to 
+register a custom Url protocol on the user's operating system: the process for doing this is
+different on each platform. 
+
+## Setting up an App with a custom Url protocol
+The custom Url protocol defines a base url which will be handled by the App itself rather than a browser.
+
+### Register your App online
+The 1st step is to register a new app by visiting: 
+
+https://developers.arcgis.com/applications/#/c4aafa91face4b7db12400e1d18a04fd/auth/
```

Comment:
can you update this to "The first step is to register a new app by visiting the [Developers page](http://links.esri.com/qtRegisterNewApp)", or something like that?